etc/scdaq/scdaq.conf

```diff
@@ -59,9 +61,10 @@ doZS:no
 ##
 ################################################################################
 
-output_filename_prefix:scout_pass_through
+output_filename_prefix:scout_GMT
```

Comment:
Same comment here wrt commenting it out.

Reply:
ok as well

Reply:
done